### PR TITLE
Schema improvements

### PR DESCRIPTION
**Pull request by Eomm**
Opening PR as a draft since on Windows tests are red

### CODE REVIEW

**Review by rodion-arr — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 When an author releases an Open Source package there are many different levels
 of support they may intend to provide.  The [Node.js Package Maintenence Working Group](https://github.com/nodejs/package-maintenance)
-is working to propose [a spec](https://github.com/nodejs/package-maintenance/issues/220) to help package authors declare their intended support goals.  This package provides
+is working to propose [a spec](https://github.com/nodejs/package-maintenance/blob/master/docs/PACKAGE-SUPPORT.md) to help package authors declare their intended support goals.  This package provides
```
Comment (ljharb) [inline README.md]:
```suggestion
is working to propose [a spec](https://github.com/nodejs/package-maintenance/blob/HEAD/docs/PACKAGE-SUPPORT.md) to help package authors declare their intended support goals.  This package provides
```

File: lib/cli/index.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 const yargs = require('yargs/yargs');
 const path = require('path');
-const Loggerr = require('loggerr');
+const { Loggerr } = require('loggerr');
```
Comment (ljharb) [inline lib/cli/index.js:4]: gross, why would they make this package less ergonomic in a prerelease

**Review by ljharb — APPROVED** [review]
pending comments

File: lib/cli/index.js (hunk 2 above)
Comment (Eomm) [reply lib/cli/index.js:4]: I read the source code to fix it

Now `require('loggerr')` return a singleton and the prop return the constructor

**Comment by rodion-arr** [comment]
Can we land this please? I'd like to submit a PR and it is required to do same change on Logger..

**Comment by Eomm** [comment]
It is ok for me

I don't remember if there is any minimal approval before the merge

**Comment by rodion-arr** [comment]
Here is the example of policy from `wiby` - https://github.com/pkgjs/wiby/pull/22#issuecomment-668818601
I assume it is for all organization?

**Comment by Eomm** [comment]
Oh found [here](https://github.com/nodejs/package-maintenance/blob/master/Governance.md#landing-prs-3) and it is the same you posted!

Merging!

**Pull request merged**